pwa/src/templates/templateParts/searchComponent/SearchComponentTemplate.tsx

```diff
@@ -23,7 +23,44 @@ export const SearchComponentTemplate: React.FC<SearchComponentTemplateProps> = (
   } = useForm();
 
   const onSubmit = (data: any): void => {
-    setFilters({ ...filters, search: data.name });
+    setFilters({
+      ...filters,
+      search: data.name,
+      softwareType: "",
+      developmentStatus: "",
+      platforms: [],
+      category: "",
+      "nl.commonground.layerType": [],
+      "nl.gemma.bedrijfsfuncties": [],
+      "nl.gemma.bedrijfsservices": [],
+      "nl.gemma.referentieComponenten": [],
+      "nl.gemma.applicatiefunctie": "",
+      "nl.upl": [],
+      "maintenance.type": "",
+      "legal.license": "",
+      "legal.mainCopyrightOwner": "",
+    });
+    navigate("/components");
+  };
+
+  const clearFilters = () => {
+    setFilters({
+      ...filters,
+      search: "",
+      softwareType: "",
+      developmentStatus: "",
+      platforms: [],
+      category: "",
+      "nl.commonground.layerType": [],
+      "nl.gemma.bedrijfsfuncties": [],
+      "nl.gemma.bedrijfsservices": [],
+      "nl.gemma.referentieComponenten": [],
+      "nl.gemma.applicatiefunctie": "",
+      "nl.upl": [],
+      "maintenance.type": "",
+      "legal.license": "",
+      "legal.mainCopyrightOwner": "",
+    });
```

Comment:
I think that the following does the same. A lot of properties in this object are optional, no need to set them with empty values 😄 

```
setFilters({
    resultDisplayLayout: filters.resultDisplayLayout,
    dependenciesDisplayLayout: filters.dependenciesDisplayLayout,
    currentPage: filters.currentPage,
} as IFilters);
```

Reply:
Note: you can do the same at the onSubmit, but then also add the search param, like so:

```
setFilters({
    search: data.name,
    resultDisplayLayout: filters.resultDisplayLayout,
    dependenciesDisplayLayout: filters.dependenciesDisplayLayout,
    currentPage: filters.currentPage,
} as IFilters);
```